bots/faq_bot.py

```diff
@@ -18,6 +20,6 @@ async def on_members_added_activity(
     async def on_message_activity(self, turn_context: TurnContext):
         return await turn_context.send_activity(
             MessageFactory.text(
-                self.qa_model.find_best_answer(turn_context.activity.text)
+                self.qa_model.find_best_answer(CONFIG.CONTEXT_GENERAL, turn_context.activity.text)
```

Comment:
so this PR will make it currently only answer questions from general right?

Reply:
code wise yes, but in practice, the general spreadsheet has the same content as the as the gc specialized sheet. so people interacting with the bot will still see all answers, until we split the content